src/main.ts

```diff
@@ -0,0 +1,5 @@
+import 'source-map-support/register';
```

Comment:
I think this is used to make sure exceptions have the proper stack trace. Does this work for typescript as well?

Reply:
> I think this is used to make sure exceptions have the proper stack trace. Does this work for typescript as well?

@jiahaog yes it does. TS generates the sourcemaps (because `tsconfig.json` says `sourceMap: true`), then it's all the same for `source-map-support` 🙂.